src/cache/cache_las.c

```diff
@@ -827,13 +823,16 @@ __wt_las_cursor_position(
          * There may be no block of lookaside entries if they have been
```

Comment:
There aren't blocks anymore, maybe edit this comment to say new entries can be inserted in between.